gax/BUILD.bazel

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Google Inc.  All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+cc_library(
+    name = "cpp_retry_policy",
```

Comment:
Does this require a dedicated cc_library build rule? What about a single gax rule, that initially contains only this header but to which we will add the additional parts of gax?

Reply:
This cc_library rule is a direct result of cargo cult programming/not being familiar with bazel. There's currently just the header, and my WIP backoff policy change is also header only. I'll bug @vam-google for help correctly setting up the rules.